src/cirrus/builtins/cloudformation/resources.yml

```diff
@@ -14,6 +14,12 @@ Resources:
     Properties:
       TopicName: "#{AWS::StackName}-publish"
 
+  # SNS Topic for failed workflows
+  WorkflowEventTopic:
+    Type: "AWS::SNS::Topic"
+    Properties:
+      TopicName: "#{AWS::StackName}-workflow-event"
```

Comment:
everywhere else this is named as `workflow-event-topic` instead of `workflow-event`